pkg/controller/sync/dispatch/unmanaged.go

```diff
@@ -111,7 +116,7 @@ func (d *unmanagedDispatcherImpl) RemoveManagedLabel(clusterName string, cluster
 
 		util.RemoveManagedLabel(updateObj)
 
-		_, err := client.Resources(updateObj.GetNamespace()).Update(updateObj, metav1.UpdateOptions{})
+		err := client.Update(context.Background(), updateObj)
```

Comment:
Do we not need to set GVK here because we're dealing with a cached informer object that should already have it set?

Reply:
gvk is only required for operations where an object isn't sourced from the informer or derived from a federated resource (i.e. get and delete). Otherwise apiVersion will be populated (as evidenced by ci running without error).

Reply:
I believe that is correct, based on my reading.